python/lsst/meas/extensions/psfex/psfex.py

```diff
@@ -796,7 +801,7 @@ def read_samplesLsst(prefs, set, filename, frmin, frmax, ext, next, catindex, co
 
     edgeBit = [k for k, v in getFlags().items() if v == "edge"][0]
 
-    for i, xc, yc in zip(range(nobj), xm, ym):
+    for i, xc, yc in zip(list(range(nobj)), xm, ym):
```

Comment:
I wasn't sure about that one because I don't know enough about how zip works. So it *is* ok to use an iterator in zip?